config/jobs/kubernetes-sigs/e2e-framework/e2e-framework-presubmits.yaml

```diff
@@ -6,11 +6,15 @@ presubmits:
     path_alias: sigs.k8s.io/e2e-framework
     spec:
       containers:
-      - image: gcr.io/k8s-staging-releng/releng-ci:latest-go1.16
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210615-c973edd-master
         imagePullPolicy: Always
         command:
+        - runner.sh
```

Comment:
for this job we can keep the previous image, this is running lint and other checks, we don't need to have special things for this